kobo/apps/audit_log/models.py

```diff
@@ -1,10 +1,22 @@
+import logging
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
```

Comment:
FYI: you can import this instead:

`from kobo.apps.kobo_auth.shortcuts import User`